docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+
+```
+$ gcloud compute instances create {INSTANCE_NAME} \
+  --confidential-compute \
+  --shielded-secure-boot \
+  --maintenance-policy Terminate \
+  --scopes cloud-platform \
+  --image-project confidential-space-images \
+  --image-family confidential-space \
+  --service-account {SERVICE_ACCOUNT} \
+  --metadata ^~^tee-image-reference=ghcr.io/iabtechlab/uid2-operator@sha256:{IMAGE_SHA}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=integ~tee-env-API_TOKEN={API_TOKEN}
+```
+
+(**GWH_YS11 Is there any info we can provide re the results of running the above, how they can make sure it was successful?**)
+
+### Production Deployment
+
+To deploy a new UID2 Operator in the GCP Confidential Space Enclave into the integration environment, follow these steps.
+
+(**GWH_YS12 if we're not including the Integ steps I should put all the steps here.**)
+
+You will be provided a new `{API_TOKEN}`, and `~tee-env-DEPLOYMENT_ENVIRONMENT=integ~` needs to be changed to
+`~tee-env-DEPLOYMENT_ENVIRONMENT=prod~`. (**GWH_YS13 re "you will be provided a new API token... when they request the account are they given both? If not, when should they request the prod API token? That isn't covered.**)
+
+We recommend that you also specify the machine type in the gcloud script. The default value is `n2d-standard-2`; we recommend that you run the UID2 operator on a machine type of `n2d-standard-16`. (**GWH_YS14 is machine type something they might need to set up as part of the Google account setup? If so we should specify this info re machine type in the earlier section.**)
+
+The following example of the production deployment script uses placeholder values:
+
+```
+$ gcloud compute instances create {INSTANCE_NAME} \
+  --machine-type n2d-standard-16 \
+  --confidential-compute \
+  --shielded-secure-boot \
+  --maintenance-policy Terminate \
+  --scopes cloud-platform \
+  --image-project confidential-space-images \
+  --image-family confidential-space \
+  --service-account {SERVICE_ACCOUNT} \
+  --metadata ^~^tee-image-reference=ghcr.io/iabtechlab/uid2-operator@sha256:{IMAGE_SHA}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=prod~tee-env-API_TOKEN={API_TOKEN}
+```
+
+>NOTE: This script includes an additional option not used in the script for the integration environment, `--machine-type n2d-standard-16`. By specifying this value you can make sure that the production deployment of UID2 Operator runs on the recommended machine type that matches the production configuration.
+
+## Upgrade
+
+When a new version of UID2 Google Cloud Platform Confidential Space is released, private operators are informed via email. There is a window of time for upgrade, after which the older version is deactivated and is no longer supported.
+
+If you're upgrading to a new version, update the `{IMAGE_SHA}` in your configuration ([Integration Deployment](#integration-deployment) and/or [Production Deployment](#production-deployment)) to the new value.
```

Comment:
That's not sufficient. What should you do with the already running instances? How do you switch over?

Reply:
Probably we should suggest them to use terraform to maintain their deployments.
In that way they only need to update the {IMAGE_SHA}, and GCP instance group manager will handle the rolling update.

Reply:
TBD